README.rst

```diff
@@ -170,6 +170,157 @@ EdgePopulation
    >>> selection = population.connecting_edges([1, 2, 3], [4, 5, 6])
 
 
+Reports
+~~~~~~~
+
+SpikeReader
++++++++++++
+
+.. code-block:: pycon
+
+   >>> import libsonata
+
+   >>> spikes = libsonata.SpikeReader('path/to/H5/file')
+
+   # list populations
+   >>> spikes.get_populations_names()
+
+   # open population
+   >>> population = spikes['<name>']
+
+
+SpikePopulation
++++++++++++++++
+
+.. code-block:: pycon
+
+   # get all spikes [(node_id, timestep)]
+   >>> population.get()
+   [(5, 0.1), (2, 0.2), (3, 0.3), (2, 0.7), (3, 1.3)]
+
+   # get all spikes betwen tstart and tstop
+   >>> population.get(0.2, 1.0)
```

Comment:
Maybe better to advertise the named argument instead (less error prone) ?

```
population.get(tstart=0.2, tstop=1.0)
```
Also as I said in : 
https://github.com/BlueBrain/libsonata/pull/96

There is a bunch of implicit behavior we need to specify here: 

```
population.get(tstart=-1, tstop=0.5)
```
Negative values for times stand for minimum report time for tstart and maximum report time for tstop.
Also `node_ids=[]` returns all node_ids.
So : 
```
population.get() == population.get(tstart=-1, tstop=-1, node_ids=[])
```

Reply:
> Maybe better to advertise the named argument instead (less error prone) ?
> 
> ```
> population.get(tstart=0.2, tstop=1.0)
> ```
I totally agree, this way is much clearer 

> Also as I said in :
> #96
> 
> There is a bunch of implicit behavior we need to specify here:
> 
> ```
> population.get(tstart=-1, tstop=0.5)
> ```
> 
> Negative values for times stand for minimum report time for tstart and maximum report time for tstop.
> Also `node_ids=[]` returns all node_ids.
> So :
> 
> ```
> population.get() == population.get(tstart=-1, tstop=-1, node_ids=[])
> ```

I am not sure if explaining this in the readme is the best place, I think we should keep this simple here and maybe extend the documentation for the more complicated cases somewhere else?

Reply:
For snap we have something like this : 
https://bluebrainsnap.readthedocs.io/en/latest/submodules/bluepysnap.nodes.html#bluepysnap.nodes.NodePopulation.ids

So all the possible inputs are listed here and this draw directly from the docstrings. 

What is cool about that is we also have this doc when you do in your python console: 
```
help(NodePopulation.ids)
```
So I am not sure what s the best place for this can you check with @haleepfl  ?

Reply:
This type of documentation should definitely be in the docstrings. It is best to have the documentation close to the source, and gives it the best chance of staying up-to-date if the behavior changes in the future. This is also applicable for the C++ doxygen strings.